### PR TITLE
Platform manual onboarding

### PR DESCRIPTION
**Pull request by Daxesh25**

### CODE REVIEW

**Comment by truclk** [comment]
I have two feedback:
- For the screen recording. 
https://lh7-us.googleusercontent.com/89Q2Nu6V1qFpI6tj2li1vAo8r_GjlzIh2kjkJIcAgCpMviS6PEdHyGfzA2LVZusY484UBJ7JaNBs7-pr5MJDhFGCc3_ag9DTn4RocZ9Q1WCnAkRfTxYxQtT0ziDJlCgRMwrNjiebJx6Fd7_bcAbKSaA
We might need to hide the bookmark bar and keep only the visible part of the website for professional reference.
- Second it should be upload into S3 for storage, please don't use google drive because it was not suppose to use on website. You can ask DevOps team for helping with the upload

File: pages/GettingStarted/Onboarding/onboarding-aws-with-manual-setup.md [hunk 1]
```diff
@@ -106,19 +107,19 @@ AWS needs a place to save your cost and usage report. In this step, you will cre
...
 3.  Click **Save**.
 4.  In the **Report Delivery Options** details:
-    ![](https://nops-docs-img.s3.amazonaws.com/gettingstarted/gs-aws-cur-delivery-details.png)
+    ![](https://lh7-us.googleusercontent.com/YYv6FnRSyooyV_6lsMXwguJHr5EkyksDGvi6dzFgyfcw520hF7prDfZIuBYS4s_zsmP9tiT07IG69C4XxUM7LL7CBOrwnyLSTcnzr1DbYj8Rqd_2yG9B1-5zzJyDNjH4XqX0I_C0SGT8yYU9MEn2DIU)
 
 
     * Enter the **report path prefix** (required) – Suggestion: _nopsbilling_
-    * Choose **Daily** (mandatory) for **Time granularity**.
+    * Choose **Hourly** (mandatory) for **Time granularity**.
     * Select an option for **Report versioning** (optional) — Suggestion: Overwrite existing report.
```
Comment (truclk) [inline pages/GettingStarted/Onboarding/onboarding-aws-with-manual-setup.md:121]: It's mandatory now. No suggestion, we can't handle things with new version

File: pages/Integrations_and_Agents/Integrations/Configure-nOps-Kubernetes-Agent-on-EKS.md [hunk 2]
```diff
@@ -59,3 +59,8 @@ And to execute:./script\_name.sh
 
 After successfully configuration, you gain complete visibility into your workloads, tailored to your preferred level of detail. While your EKS expenditure typically remains opaque, pinpointing cloud inefficiencies can be a challenge. nOps streamlines this process by automatically detecting wastage through CPU and memory metrics, empowering you to optimize resources and realize immediate cost savings.
 
+**FAQs**
+
+1. I have prometheus installed and I use it for a variety of purposes in my cluster, will this affect that?
+ **Ans:** No, our agent leverages it's own prometheus in it's own namespace so that doesn't affect your current deployments.
```
Comment (truclk) [inline pages/Integrations_and_Agents/Integrations/Configure-nOps-Kubernetes-Agent-on-EKS.md:65]: The sentence not correct 100%. It should export data the same as our requirement.

**Review by truclk — CHANGES_REQUESTED** [review]

File: pages/Integrations_and_Agents/Integrations/Configure-nOps-Kubernetes-Agent-on-EKS.md (hunk 2 above)
Comment (truclk) [reply pages/Integrations_and_Agents/Integrations/Configure-nOps-Kubernetes-Agent-on-EKS.md:65]: @andremilk Can you help us on finish the Answer: which metrics or exporter is require for this manual provided Prometheus

Comment (andremilk) [reply pages/Integrations_and_Agents/Integrations/Configure-nOps-Kubernetes-Agent-on-EKS.md:65]: Maybe for clarity we should rephrase this...
Ans: No, the agent installation process deploys it's own Prometheus with it's own name space so there's no interference with current Prometheus deployments.

**Review by nopsgrego — APPROVED** [review]

**Comment by nopsgrego** [comment]
merging per Truc's approval in Slack

**Pull request merged**